test/built-ins/Proxy/apply/trap-is-missing.js

```diff
@@ -2,18 +2,22 @@
 // This code is governed by the BSD license found in the LICENSE file.
 /*---
 es6id: 9.5.13
+esid: sec-proxy-object-internal-methods-and-internal-slots-call-thisargument-argumentslist
 description: >
-    If trap is undefined, propagate the call to the target object.
+    If trap is null or undefined, propagate the call to the target object.
 info: >
     [[Call]] (thisArgument, argumentsList)
 
     7. If trap is undefined, then Return Call(target, thisArgument,
     argumentsList).
 ---*/
 
-var target = function(a, b) {
+function target(a, b) {
+    assert.sameValue(this, ctx);
     return a + b;
-};
-var p = new Proxy(target, {});
+}
 
-assert.sameValue(p(1, 2), 3);
+var ctx = {};
+var p = new Proxy(target, {});
+var res = p.call(ctx, 1, 2);
+assert.sameValue(res, 3, "`apply` trap is missing");
```

Comment:
Instead of modifying the existing test file (which tests the implicit `apply` trap path), this should be added as a new test file.